docs/local-kiln.md

```diff
@@ -0,0 +1,43 @@
+---
+id: local-kiln
+title: Running Kiln Locally
+sidebar_label: Running Kiln Locally
+---
+
+To develop Kiln you need to run it locally. If you want to use the Vue Development Tools plugin in your browser, you will also need to run it locally because the Vue Development Tools only work when Vue is in development mode and Kiln is only in development mode when it is run locally.
+
+### Steps to run it locally
+
+1. Clone the [Clay Kiln GitHub Repository](https://github.com/clay/clay-kiln)
+2. cd into newly cloned clay-kiln directory and run the command `npm link`
+
+---
+The next 2 steps are only neccessary if you are actively developing Clay-Kiln, not if you just want a local version linked so you can use the Vue dev tools in your browser.
```

Comment:
Maybe there could be another section "Development of Kiln" or at least a heading that indicates that this is where to find the dev instructions.